__tests__/seed.test.js

```diff
@@ -1,64 +1,72 @@
+  db.end();
+});
 
 describe("GET /api/topics", () => {
-    test("responds with an array of topic objects", () => {
-        return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then(({body}) => {
-            expect(body).toHaveLength(3)
-            body.forEach((topic) => {
-                expect(topic).toMatchObject({ description: expect.any(String), slug: expect.any(String) })
-            })
-        })
-    })
-        // .then(({body}) => {
-        //     expect(body).toHaveLength(3)
-        //     expect(body).toMatchObject([
-        //         {
-        //           description: 'The man, the Mitch, the legend',
-        //           slug: 'mitch'
-        //         },
-        //         {
-        //           description: 'Not dogs',
-        //           slug: 'cats'
-        //         },
-        //         {
-        //           description: 'what books are made of',
-        //           slug: 'paper'
-        //         }
-        //       ])
-        // })
+  test("responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveLength(3);
+        body.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
 
-    describe("GET /api/topics", () => {
-        test("responds with 404 when incorrect spelling", () => {
-            return request(app)
-            .get("/api/incorrectspelling")
-            .expect(404)
-        })
-    })
+  describe("GET: 404", () => {
+    test("responds with 404 when incorrect spelling", () => {
+      return request(app).get("/api/incorrectspelling").expect(404);
+    });
+  });
 
-describe("GET /api", () => {
+  describe("GET /api", () => {
     test("responds with object describing all available endpoints", () => {
-        return request(app)
-        .get ("/api")
+      return request(app)
+        .get("/api")
         .expect(200)
-        .then(({body}) => {
-            expect(body).toEqual(endPoints)
-        })
-    })
-})
-})
+        .then(({ body }) => {
+          expect(body).toEqual(endPoints);
+        });
+    });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("responds with an article object with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+  test("GET:400, responds with status code and message when given invalid id", () => {
```

Comment:
i would describe this as a valid but non-existent id -so we should get 404 not found, we cant find an article with this id as it doesnt exist. an invalid id would be e.g. /api/articles/banana. What should we see then? We need a separate test for that